llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -241,8 +241,6 @@ def VLOpFrag : PatFrag<(ops), (XLenVT (VLOp (XLenVT AVL:$vl)))>;
 // This must be kept in sync with RISCV::VLMaxSentinel.
 def VLMax : OutPatFrag<(ops), (XLenVT -1)>;
 
-def SelectFPImm : ComplexPattern<fAny, 1, "selectFPImm", [], [], 1>;
```

Comment:
Doesn't this need to be moved to VVLPatterns.td?

Reply:
Whoops sorry, didn't notice it came from there